plugins/sftpput/ui/src/main/java/org/pentaho/di/ui/trans/steps/sftpput/SFTPPutDialog.java

```diff
@@ -206,7 +210,7 @@ public class SFTPPutDialog extends BaseStepDialog implements StepDialogInterface
 
   private SFTPClient sftpclient = null;
 
-  public SFTPPutDialog( Shell parent, Object in, TransMeta tr, String sname ) {
+  public SFTPPutDialog(Shell parent, Object in, TransMeta tr, String sname ) {
```

Comment:
Please fix the spacing according to pentaho-coding-standards. 
Refer https://github.com/pentaho/pentaho-coding-standards

Reply:
Fixed the issue